src/test/scala/com/mozilla/telemetry/streaming/TestErrorAggregator.scala

```diff
@@ -449,4 +449,18 @@ class TestErrorAggregator extends AsyncFlatSpec with Matchers with BeforeAndAfte
     val df2 = ErrorAggregator.aggregate(spark.sqlContext.createDataset(messages).toDF, raiseOnError = false, online = false)
     df2.where("build_id IS NULL").collect().length should be (0)
   }
+
+  "display version" can "be undefined" in {
+    import spark.implicits._
+    val messages = TestUtils.generateMainMessages(
+      1, None, None, List[String]("displayVersion")
```

Comment:
nit: you don't have to specify the `String` type here, since it can infer it from the function definition. So you can use either `List("displayVersion")`, or my personal favorite, `"displayVersion" :: Nil`

Reply:
👍